shard.yml

```diff
@@ -1,2 +1,7 @@
 name: pg
 version: 0.11.0
+
+dependencies:
+  db:
+    github: crystal-lang/crystal-db
+    branch: master
```

Comment:
@asterite it should be `version: ~> 0.2.2`. So crystal-db/master can move forward without breaking will/crystal-pg master.

Or the releases of crystal-pg should be on a branch where the db version is fixed, but that is too much burden.

As soon as we crystal-pg onto db I will move forward with [unprepared statements](https://github.com/crystal-lang/crystal-db/pull/25) / [transactions](https://github.com/crystal-lang/crystal-db/tree/feature/transactions) on all 3 drivers :-)